client/package.json

```diff
@@ -49,6 +50,8 @@
     "@types/react-router-dom": "^4.3.4",
     "@types/reactstrap": "^8.0.1",
     "@types/redux": "^3.6.0",
+    "@types/redux-form": "^8.1.4",
+    "@types/yup": "^0.26.22",
```

Comment:
I believe Yurii deleted yup 
now u add it


could u pls explain it?

Reply:
My mistake. Deleted.